operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml

```diff
@@ -9178,6 +9180,10 @@ data:
         - name: {{ $key }}
           value: "{{ $value }}"
         {{- end }}
+        {{- range $key, $value := .Values.global.proxyEnvVars }}
```

Comment:
we already have this, see immediate line above

Reply:
Sure. But I had a hard time to set it on istioctl:
out/linux_amd64/istioctl manifest apply --set values.global.tag=$TAG \
--set proxyConfig.proxyMetadata.foo=bar
doesn't work. Do you know how to set it?

Reply:
@ostromart

Reply:
The section above works. I'll remove this part.